utilities/pipelines/staticValidation/global.module.tests.ps1

```diff
@@ -2,17 +2,19 @@
 
 param (
```

Comment:
Open question: Should we rename this file? For example to `static.module.tests.ps1` or just `module.tests.ps1`. We used to have 'global' when we still had tests inside of modules - that that's not the case anymore...

Reply:
I'd vote for 'module.tests.ps1'

Reply:
module.pester.tests.ps1?

Reply:
Isn't *.tests.ps1 considered pester by default unless explicitly customised?

Reply:
True, then module.tests.ps1 works for me if we all agree.

Reply:
Sounds good to me :)

Reply:
Renamed